src/main/java/org/openrewrite/github/UseGradleWrapper.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2024 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.github;
+
+import org.openrewrite.*;
+import org.openrewrite.yaml.JsonPathMatcher;
+import org.openrewrite.yaml.YamlIsoVisitor;
+import org.openrewrite.yaml.tree.Yaml;
+
+import javax.annotation.Nullable;
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class UseGradleWrapper extends Recipe {
```

Comment:
Great start! I feel this should maybe be a ScanningRecipe that first checks if any gradle wrapper is present before we make changes. The accumulator could be an `AtomicBoolean wrapperFound`; the scanner would merely set that to true if indeed found. Would you agree there?

Reply:
Makes sense. Added this in ce3bcc9